rules/integrations/aws/ml_cloudtrail_rare_error_code.toml

```diff
@@ -23,19 +23,81 @@ interval = "15m"
 name = "Rare AWS Error Code"
-note = """
-## Config
+note = """## Triage and analysis
 
-The AWS Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule.
+### Investigating Rare AWS Error Code
+
+CloudTrail logging provides visibility on actions taken within an AWS environment. By monitoring these events and
+understanding what is considered normal behavior within an organization, you can spot suspicious or malicious activity
+when deviations occur.
+
+This rule uses a machine learning job detected an unusual error in a CloudTrail message. These can be byproducts of
+attempted or successful persistence, privilege escalation, defense evasion, discovery, lateral movement, or collection.
+
+Detection alerts from this rule indicate a rare and unusual error code that was associated with the response to an AWS
+API command or method call.
+
+#### Possible investigation steps
+
+- Examine the history of the error, and if the error, which is visible in the `aws.cloudtrail.error_code field`, only
+manifested recently, it might be related to recent changes in an automation module or script.
+- Investigate other alerts associated with the user account during the past 48 hours.
+- Validate the activity is not related to planned patches, updates, or network administrator activity.
+- Examine the request parameters. These may provide indications as to the source of the program or the nature of the
+task being performed when the error occurred.
+    - Check whether the error is related to unsuccessful attempts to enumerate or access objects, data, or secrets.
+- Considering the source IP address and geolocation for the calling user who issued the command:
+    - Do they look normal for the calling user?
+    - If the source is an EC2 IP address, is it associated with an EC2 instance in one of your accounts, or could
+    it be sourcing from an EC2 instance that's not under your control?
+    - If it is an authorized EC2 instance, is the activity associated with normal behavior for the instance role or roles?
+    Are there any other alerts or signs of suspicious activity involving this instance?
+- Consider the time of day. If the user is a human, not a program or script, did the activity take place during a normal
+time of day?
+- Contact the account owner and confirm whether they are aware of this activity if suspicious.
+- If the account is suspected to have been compromised, track access to servers, services, and data accessed by the
+account in the last 24 hours to scope potentially compromised assets.
+
+### False positive analysis
 
-## Triage and analysis
+- Examine the history of the command. If the command, which is visible in the `event.action field`, only manifested
+recently, it might be part of a new automation module or script. If it has a consistent cadence (for example, if it
+appears in small numbers on a weekly or monthly cadence), it might be part of a housekeeping or maintenance process.
```

Comment:
```suggestion
appears in small numbers on a weekly or monthly cadence), it might be part of a housekeeping or maintenance process. You can find the command in the `event.action field` field.
```